specs/chain/hs/src/Cardano/Spec/Consensus/Block.hs

```diff
@@ -56,3 +56,18 @@ instance Block CBM.Block where
 
   bHeader b = b ^. CBM.bHeader
   bBody   b = b ^. CBM.bBody
+
+
+-- | Turns a generic block to a concrete 'CBM.Block'
+concretiseBlock :: Block b => b -> CBM.Block
```

Comment:
As suggested by @dcoutts, we use the term "elaborate" when going from abstract to concrete. I don't know if we want to use this convention here as well.

Reply:
If I am not mistaken, that was a different context, hence the terminology guideline does not apply here. Here we already have actual blocks and this is just about translating to another specific block type.